tests/web/rpc/v1/test_endpoint_time_series/test_endpoint_time_series.py

```diff
@@ -1277,3 +1281,52 @@ def use_eap_items_table(
     ) -> None:
         snuba_set_config("use_eap_items_table", True)
         snuba_set_config("use_eap_items_table_start_timestamp_seconds", 0)
+
+    def test_preflight(self) -> None:
+        # store a a test metric with a value of 1, every second of one hour
+        granularity_secs = 15
+        query_duration = granularity_secs * 1
+        store_spans_timeseries(
+            BASE_TIME,
+            1,
+            query_duration,
+            metrics=[DummyMetric("test_preflight_metric", get_value=lambda x: 1)],
+        )
+
+        preflight_message = TimeSeriesRequest(
```

Comment:
this test should run:

1. The non-sampled request
2. the sampled request

and then compare that (2) has fewer elements than 1 by at least a factor of 100